boot/boot.js

```diff
@@ -287,6 +287,136 @@ $tw.utils.decodeURIComponentSafe = function(s) {
 	return v;
 };
 
+/*************
+Code to make the permalink/permaview URLs more readable.
+(These comments are just to help reviewers; they will eventually be removed.)
+***************/
+
+
+/*************
+Helpers for encoding/decoding URIs.  **Note** that these are throw-away, only
+used to temporarily build `charMatch`, `spaceMatch`, `sentenceEnding`, and
+`sentenceTrailing`, all of which will eventually be hard-coded.
+***************/
+
+/*
+The character that will substitute for a space in the URL
+*/
+var SPACE = "_";
```

Comment:
IMO the name should be `SUBST_SPACE`, `SEPARATOR` or something similar
SPACE feels wrong because it's a char other than space.